assets/htmlTemplates/abusers.html

```diff
@@ -23,6 +23,11 @@
                 {{ end }}
             </select>
         </div>
+        <div class="form-group">
+            <label for="selector">Selector</label>
+            <br>
+            <select id="" name="selector"><option value="sender_domain">Sender domain</option><option value="sasl_username">Sasl username</option></select>
```

Comment:
oh I generate html with emmet, which doesnt like new lines and gives an empty id, will fix